.gitattributes

```diff
@@ -5,4 +5,24 @@
 *.jpg binary
 
 #Files we do not want to distribute
-/tests export-ignore
\ No newline at end of file
+.ci/                 export-ignore
+.editorconfig        export-ignore
+.gitattributes       export-ignore
+.github/             export-ignore
+.gitignore           export-ignore
+.php_cs.dist         export-ignore
+.travis.yml          export-ignore
+appveyor.yml         export-ignore
+box.json.dist        export-ignore
+build/               export-ignore
+CHANGELOG.md         export-ignore
+codecov.yml          export-ignore
+composer.lock        export-ignore
+devTools/            export-ignore
+infection.json.dist  export-ignore
+LICENSE              export-ignore
+Makefile             export-ignore
+phpunit.xml.dist     export-ignore
+README.md            export-ignore
+tests/               export-ignore
+setup_environment.sh export-ignore
```

Comment:
Maybe sort

- by type (directories first)
- then alphabetically 

?

Reply:
@raphaelstolt sorry for bothering you, are you going to finish this PR or can we take over?

Reply:
Feel free to sort the ignored artifacts to your liking and commit them in this PR.